tuna/miopen/subcmd/export_db.py

```diff
@@ -456,6 +457,13 @@ def export_pdb(dbt, args):
       ins_cfg_id = cfg_map[cfg_entry.id]
     else:
       cfg_dict = get_cfg_dict(cfg_entry, cfg_entry.input_t)
+
+      #override cfg layout with fdb key layout
+      if perf_db_entry.fdb_key:
```

Comment:
are we assuming that we imported configs with the wrong layout and so we overwrite it with the fdb key layout?

Reply:
There was a fin issue which was causing layouts other than the one specified by the config to be tuned.
changing this statement to raise an error instead.